cmd/dep/root_analyzer.go

```diff
@@ -30,22 +30,25 @@ type importer interface {
 //   then external tools.
 type rootAnalyzer struct {
 	skipTools  bool
+	gopath     bool
 	ctx        *dep.Ctx
 	sm         gps.SourceManager
 	directDeps map[string]bool
 }
 
-func newRootAnalyzer(skipTools bool, ctx *dep.Ctx, directDeps map[string]bool, sm gps.SourceManager) *rootAnalyzer {
+func newRootAnalyzer(skipTools bool, gopath bool, ctx *dep.Ctx, directDeps map[string]bool, sm gps.SourceManager) *rootAnalyzer {
 	return &rootAnalyzer{
 		skipTools:  skipTools,
+		gopath:     gopath,
 		ctx:        ctx,
 		sm:         sm,
 		directDeps: directDeps,
 	}
 }
 
 func (a *rootAnalyzer) InitializeRootManifestAndLock(dir string, pr gps.ProjectRoot) (rootM *dep.Manifest, rootL *dep.Lock, err error) {
-	if !a.skipTools {
+	// Use importer tools only in gopath mode
+	if a.gopath && !a.skipTools {
```

Comment:
This doesn't match the behavior outlined in the [updated command spec](https://docs.google.com/document/d/1Qwa3jDKDh45t5qAGjRpeu3ZsAbkukIYDDmYJdMxH2uY)

> If -gopath is passed, then when the legacy tool import is complete (or skipped, in the event that -skip-tools is also passed), an additional search pass should be made through the current (as inferred from the cwd) GOPATH

As I read it, `-gopath` and `-skip-tools` are completely independent.